engines/dfc_provider/app/services/social_media_builder.rb

```diff
@@ -12,10 +12,16 @@ def self.social_medias(enterprise)
   def self.social_media(enterprise, name)
     return nil unless name.in?(NAMES)
 
-    url =  enterprise.attributes[name]
+    url = enterprise.public_send(name)
 
     return nil if url.blank?
 
+    if name == "instagram"
+      url = "https://www.instagram.com/#{url}/"
+    end
+
+    url = "https://#{url}" unless url.starts_with?(%r{https?://})
+
```

Comment:
I think that this should live in the model because that knows how it normalises the URLs. Maybe next time...

Reply:
Agreed!